UnityProject/Assets/Scripts/Managers/UpdateManager/UpdateManager.cs

```diff
@@ -12,6 +12,9 @@
 /// </summary>
 public class UpdateManager : MonoBehaviour
 {
+
+	public static float CashedDeltaTime = 0;
```

Comment:
Codacy found an issue: [Change the visibility of 'CashedDeltaTime' or make it 'const' or 'readonly'.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6392909)